### ISSUE TITLE
Reply below quote

### ISSUE THREAD

**Issue by adocampo** [RocketChat/Rocket.Chat#18546]
### Description:
This is not a bug, but a feature request.
For readability, it should be more natural to show what I'm replying below the quote, instead above, so when I'm chatting in general and then replying to someone before no one types anything, when quoting, the quote will show first and then my answer, then my answer won't mix with the former content... 
There are some times I'm speaking of the "subject A", then I quote someone who's asking me for the "subject B". As for now, my sentences on the "subject A" and "subject B" aren't divided by the quote, so the last sentences doesn't make sentence for the reader until he or she reads the quote _below_ the nonsense sentence. Is a lost of time for everyone.

On the other hand, when someone, let's say _Bob_ says something, then _Alice_ quotes _Bob_ and answer him, and then _Ted_ quotes _Alice_ answering _Bob_ and answer her... a fourth reader, let's say _Carol_ doesn't understand a single word until she reads from the bottom to top. That's unnatural in a chat.

### Steps to reproduce:

1. Speak on a general chat with more than two users, speak about cats for a while
2. Quote someone asking you what kind of car do you have and answer that quote
3. See how weird the result is.

### Expected behavior:

If the quote of the other asking you what car do you have _divides_ your chat about cats, all readers will understand with a glance, why are saying "Toyota" just after telling your cat's name is "Misha"

If someone quotes someone and a third quotes the second one, all the messages will be nested one below the other, not one avobe the other.
![image](https://user-images.githubusercontent.com/2266612/90124140-37251f00-dd60-11ea-998c-e89dc48bf51b.png)

### Actual behavior:

![image](https://user-images.githubusercontent.com/2266612/90120290-e0691680-dd5a-11ea-9a1e-88c4b50a02a4.png)

### Server Setup Information:

- Version of Rocket.Chat Server:  3.5.1
- Operating System: Debian 10
- Deployment Method: tar
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: v12.18.2
- MongoDB Version: 4.0.19

### Client Setup Information

- Desktop App or Browser Version: Any
- Operating System: Any

### Additional context

None

### Relevant logs:

None

**Comment by ankar84** [comment]
Same as https://github.com/RocketChat/feature-requests/issues/27

**Comment by urosino** [comment]
I agree it's more natural for the message to be displayed below the quote.

**Comment by ankar84** [comment]
> I agree it's more natural for the message to be displayed below the quote.

Indeed. All of us reading text from top to bottom, it's really natural.

**Comment by adocampo** [comment]
> Same as [RocketChat/feature-requests#27](https://github.com/RocketChat/feature-requests/issues/27)
> And I totally agree with you.

Didn't notice that issue... and that referees an older one, from 2015 which ask for quoting.
A year later they implemented quoting, but even they agree to quote below _later_, they still has this in their TODO list.
![image](https://user-images.githubusercontent.com/2266612/90367278-f59ab980-e068-11ea-9f9f-d8ace87410fc.png)

**Comment by ankar84** [comment]
@engelgabriel Hi, Gabriel!
Please, take a look at this issue.

**Comment by dromer** [comment]
Omg yes, we are evaluating rocket.chat at our company right now and this is a really weird choice for UI/UX
